README.md

```diff
@@ -89,7 +89,14 @@ Omit the `-run=` flag to run checks for all chains.
 > [!NOTE]
 > Your chain will be checked against the standard configuration requirements. These  are defined in the [specs](https://specs.optimism.io/protocol/configurability.html). However, these requirements are currently a draft, pending governance approval.
 
-### 5. Open Your Pull Request
+### 7. Run codegen
+This is a tool which will rewrite certain summary files of all the chains in the registry, including the one you are adding. The output will be checked in a continuous integration checks (it is required to pass):
+
+```
+sh scripts/codegen.sh
+```
+
+### 6. Open Your Pull Request
```

Comment:
It's now 4. 7. 6. instead of 4. 5.

Reply:
[0fd0931](https://github.com/ethereum-optimism/superchain-registry/pull/265/commits/0fd093155a954bd274d400ea169071e6acd4ad61)